app/services/manage_action.rb

```diff
@@ -118,6 +118,7 @@ def requires_consent?
 
   def requires_double_opt_in
     return false if @params[:consented] == false
+    return false unless page.plugins.map(&:class).include? Plugins::Petition
```

Comment:
Do we only do this for petitions? (what about surveys? or are those still petitions)

Reply:
This applies only to petitions. Surveys have their own plugin.